### PR TITLE
metals: init at 0.7.6

### PR DESCRIPTION
**Pull request by Tomahna**
###### Things done
Initialize metals package. Note that this package includes wrappers for vim, sublime and emacs written according to the package documentation. As I only use emacs, it's the only one properly tested.

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Tomahna** [comment]
@chrilves if you want to give it a try, there it is

**Review by ceedubs — APPROVED** [review]
👍 LGTM and is similar to a derivation for Metals that I've been using.

@Tomahna please feel free to also add me as a maintainer.

**Review by ceedubs — APPROVED** [review]

File: pkgs/development/tools/metals/default.nix [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{ stdenv, lib, coursier, jdk, jre, python, makeWrapper }:
```
Comment (ceedubs) [inline pkgs/development/tools/metals/default.nix]: `python` doesn't appear to be used. Is it meant to be here?

Comment (Tomahna) [reply pkgs/development/tools/metals/default.nix]: No it's an unfortunate copy/paste. I'll remove it.

**Review by ceedubs — APPROVED** [review]
🎉

**Comment by ryantm** [comment]
@GrahamcOfBorg build metals

**Pull request merged**